demos/smart_classroom_demo_gapi/main.cpp

```diff
@@ -415,18 +422,25 @@ std::map<int, int> GetMapFaceTrackIdToLabel(const std::vector<Track>& face_track
         face_track_id_to_label[cur_obj_id] = cur_label;
     }
     return face_track_id_to_label;
-}
-
-bool checkDynamicBatchSupport(const Core& ie, const std::string& device)  {
-    try  {
-        if (ie.GetConfig(device, CONFIG_KEY(DYN_BATCH_ENABLED)).as<std::string>() != PluginConfigParams::YES)
-            return false;
-    }
-    catch(const std::exception&)  {
-        return false;
-    }
-    return true;
-}
+}*/
+
+// bool checkDynamicBatchSupport(const Core& ie, const std::string& device)  {
+//     try  {
+//         if (ie.GetConfig(device, CONFIG_KEY(DYN_BATCH_ENABLED)).as<std::string>() != PluginConfigParams::YES)
+//             return false;
+//     }
+//     catch(const std::exception&)  {
+//         return false;
+//     }
+//     return true;
+// }
```

Comment:
Lots of commented code - is it required?

Reply:
Fixed.